data/build.gradle

```diff
@@ -0,0 +1,13 @@
+plugins {
+    id 'java-library'
+    id 'kotlin'
+}
+
+java {
+    sourceCompatibility = JavaVersion.VERSION_1_7
```

Comment:
@joseantoniocampillo  update the Java version

Reply:
done